redaxo/src/core/lib/packages/manager.php

```diff
@@ -478,9 +478,39 @@ public function checkPackageRequirement($packageId)
             if ('' != $requirements['packages'][$packageId]) {
                 $required_version = ' '.$requirements['packages'][$packageId];
             }
-            $this->message = $this->i18n('requirement_error_' . $package->getType(), $packageId.$required_version);
+
+            if (!rex_package::exists($packageId)) {
+                $addonId = explode('/', $packageId, 2)[0];
```

Comment:
können wir für dieses "string-parsing" eine klasse aufsetzen oder es in eine neue methode einer bestehenden klasse übernehmen?

Reply:
Habe nun `rex_package::splitPackageId` eingeführt